### PR TITLE
Block diagrams

### PR DESCRIPTION
**Pull request by fbook98**

### CODE REVIEW

File: gem_controllers/block_diagrams/stage_blocks/ext_ex_dc_cc.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+from control_block_diagram.components import Box, Connection, Point
+from control_block_diagram.predefined_components import Add, PIController, Limit
+
+
+def ext_ex_dc_cc(emf_feedforward):
```
Comment (atra94) [inline gem_controllers/block_diagrams/stage_blocks/ext_ex_dc_cc.py:5]: Please document this function

**Review by atra94 — DISMISSED** [review]
I know that this repository lacks a lot of documentation in general. However, new code should be commented.

Except for a few files there is not a single comment.

**Comment by max-schenke** [comment]
The pi_current_controller and torque_controller seem to be missing within the stage_blocks.
Therefore, the imports fail when I launch the example.py.

File: example.py [hunk 2]
```diff
@@ -1,18 +1,18 @@
 import gym_electric_motor as gem
-from gym_electric_motor import reference_generators as rg
 import gem_controllers as gc
 from gym_electric_motor.visualization import MotorDashboard
 from gym_electric_motor.physical_system_wrappers import FluxObserver
 
 
-env_id = 'Cont-CC-SCIM-v0'
+if __name__ == '__main__':
```
Comment (max-schenke) [inline example.py:6]: "env_id" got deleted and I think everything after the if clause should also be indented.

**Review by max-schenke — APPROVED** [review]
Looks good, code could use some more documentation but this can be added later.
Thanks!

**Pull request merged**